monitoring/alerts.yaml

```diff
@@ -141,7 +144,9 @@ groups:
   - alert: QuotaMetricsNotAvailable
     expr: |
       avg(avg_over_time(s3_cloudserver_quota_utilization_service_available{namespace="${namespace}",service="${service}"}[1m]))
-        < ${quotaUnavailabilityThreshold}
+        < ${quotaUnavailabilityThreshold} and
```

Comment:
why would this not be true on a properly running system (i.e. in which conditions)?

Reply:
I'm not sure I understand this comment, but I'm guessing it's related to our discussion earlier: so I am closing this comment, please feel free to reopen if needed